cmake/FindLAPACK.cmake

```diff
@@ -62,19 +65,33 @@ mark_as_advanced(LAPACK_LIBRARY)
 
 _lapack_check_is_working()
 
+# TODO: Checks?
+if(LAPACK_INCLUDE_DIR STREQUAL "" OR NOT LAPACK_INCLUDE_DIR)
+  set(LAPACK_INCLUDE_DIR "LAPACK_INCLUDE_DIR-PLACEHOLDER-FOR-EMPTY-INCLUDE-DIR")
+endif()
```

Comment:
The only case in which we currently need it is to include the headers for mkl_set_num_threads.

Reply:
Yep, I removed the TODO. The worst case is that the build will fail if the include directory isn't set properly, instead of failing during configuration time. It's a failure in any case.